INTERPRETERS plugin category no longer available in Ghidra PluginCategoryNames

Just a heads up, the `INTERPRETERS` plugin category has been removed with commit [ca9cfe9f9a](https://github.com/NationalSecurityAgency/ghidra/commit/ca9cfe9f9a) in Ghidra

```
2024-02-27T11:19:18.209+0100 [ERROR] [system.err] /home/quentin/Ghidrathon/src/main/java/ghidrathon/GhidrathonPlugin.java:37: error: cannot find symbol
2024-02-27T11:19:18.209+0100 [ERROR] [system.err]     category = PluginCategoryNames.INTERPRETERS,
2024-02-27T11:19:18.209+0100 [ERROR] [system.err]                                   ^
2024-02-27T11:19:18.209+0100 [ERROR] [system.err]   symbol:   variable INTERPRETERS
2024-02-27T11:19:18.209+0100 [ERROR] [system.err]   location: interface PluginCategoryNames
2024-02-27T11:19:18.279+0100 [ERROR] [system.err] Note: Some input files use or override a deprecated API.
2024-02-27T11:19:18.279+0100 [ERROR] [system.err] Note: Recompile with -Xlint:deprecation for details.
2024-02-27T11:19:18.279+0100 [ERROR] [system.err] 1 error
```

--- a/src/main/java/ghidrathon/GhidrathonPlugin.java
+++ b/src/main/java/ghidrathon/GhidrathonPlugin.java
@@ -34,7 +34,7 @@ import javax.swing.*;
 @PluginInfo(
     status = PluginStatus.STABLE,
     packageName = CorePluginPackage.NAME,
-    category = PluginCategoryNames.INTERPRETERS,
+    category = PluginCategoryNames.COMMON,
     shortDescription = "Python 3 Interpreter",
     description =
         "The FLARE team's open-source Python 3 interpreter console that is tightly integrated with"

